components/alert/alert.css

```diff
@@ -14,38 +28,66 @@
   margin-right: 1rem;
 }
 
-.alert-primary{
-  background-color: #93c5fd; /*300*/
-  color: #1e40af; /*800*/
+.alert--link{
+  text-decoration: underline;
+}
+
+.alert--primary{
+  background-color: var(--alert-primary-bkg); /*300*/
+  color: var(--alert-primary-color); /*800*/
+}
+
+.alert--primary i{
+  color: var(--alert-primary-icon); /*900*/
+}
+
+.alert--primary:hover{
+  cursor: pointer;
+  opacity: 0.85;
+  transition: 0.1s ease-in-out;
+}
+
+.alert--success{
+  background-color: var(--alert-success-bkg);
+  color: var(--alert-success-color);
+}
+
+.alert--success i{
+  color: var(--alert-success-icon);
 }
 
-.alert-primary i{
-  color: #1e3a8a; /*900*/
+.alert--success:hover{
+  cursor: pointer;
+  opacity: 0.85;
+  transition: 0.1s ease-in-out;
 }
 
-.alert-success{
-  background-color: #6eeb7b;
-  color: #065f46;
+.alert--warning{
+  background-color: var(--alert-warning-bkg);
+  color: var(--alert-warning-color);
 }
 
-.alert-success i{
-  color: #064e3b;
+.alert--warning i{
+  color: var(--alert-warning-icon);
 }
 
-.alert-warning{
-  background-color: #fcd34d;
-  color: #92400e;
+.alert--warning:hover{
+  cursor: pointer;
+  opacity: 0.85;
+  transition: 0.1s ease-in-out;
```

Comment:
it's good to define which property you are apply transition to 

```css
transition: background-color 0.1s linear;
```

Reply:
Did not know you can apply transition like this. Thanks for showing this!